### ISSUE TITLE
2.4.3 win7 x64 | "<your name> joined the group" message appears chaotically even if you already joined

### ISSUE THREAD

**Issue by imashxtyz** [telegramdesktop/tdesktop#8796]
It started to happen after the "anonymous admins" update and after I joined durov's chat. sometimes the chat would randomly "drop" or "jump" on the chat list, and the message "'your name' joined the group" would appear in the chat. if you do search messages for your account, you will not see it, so i guess it's only local displaying and not actual service message on the server. it only happened for durov's chat several times, i thought the problem was the speed of posting there, but now it happened for another slow group.
unfortunately i couldn't figure out how to recreate this bug, it's too unpredictable.

**Comment by Mohammed-Laith** [comment]
You said:
"_i guess it's only local displaying and not actual service message on the server._"
Your assumption is, unfortunately, wrong.
Telegram Desktop doesn't has any cache for the messages.
It only has a small cache for that while you are working on it, and when you close it, it erases such things, so no local cache of the messages exists for TD, it's not like the Android client.
And it's not something related to TD only, I was able to reproduce it on the Android client, but just on @durovschat, I wasn't able to reproduce it in any other group.
And lastly, I was able to find this service mesaage when searching for my account messages there, whether on Android's client or TD.
I just wanted to clarify some things, if anyone will read.
